.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -0,0 +1,56 @@
+---
+name: "\U0001F41EBug report"
+about: Report a bug in webhint
+---
+<!--
+
+Hi there 👋
+
+We are sorry you've found an issue in our project. Before
+opening a new issue please make sure to search in the existing
+ones (even closed!) as they might contain information about
+workarounds, resolution, or progress updates.
+
+-->
+
+# 🐛 Bug report
+
+## Description
+
+<!-- ✍️ What were you doing? -->
```

Comment:
In addition to "What were you doing?" how about something like: "What were you doing? What do we need to do to experience the issue you saw?"